fakeassignments/fakeassignments.go

```diff
@@ -3,28 +3,29 @@ package fakeassignments
 import (
 	"io/ioutil"
 	"os"
-	"os/user"
+
+	"github.com/Betterment/testtrack-cli/paths"
 
 	"gopkg.in/yaml.v2"
 )
 
 // Read reads or creates the assignment file
 func Read() (*map[string]string, error) {
-	user, err := user.Current()
+	configDir, err := paths.FakeServerConfigDir()
```

Comment:
I introduced a helper package and method for returning the directory where fake server config is stored.

This makes it easier to override this location for testing purposes. It felt like a useful addition rather than using mocks (not that I know how to mock in golang anyways).

Reply:
Nice 👍 -- this was one of the things I was thinking we might have to do if ever we want to run the go server in some fancier testing/QA environments.